deployment/permissions/service.pushnotification.pushnotificationtemplate.ini

```diff
@@ -8,3 +8,8 @@ permissionItem12.service = pushnotification_pushnotificationtemplate
 permissionItem12.action = sendCommand
 permissionItem12.partnerId = 0
 permissionItem12.permissions = WEBCAST_PRODUCER_PERMISSION
+
+permissionItem13.service = pushnotification_pushnotificationtemplate
+permissionItem13.action = sendCommand
```

Comment:
Every permission items presents an action, in this case you override sendCommand permissions, yo should add this permissions to the above item

Reply:
we tried that but it was not successful since we wanted this to happen on partner -1 and partner 0
also this will cause both permissions to work and not override